stmt/stmts/update.go

```diff
@@ -85,19 +85,19 @@ func (s *UpdateStmt) SetText(text string) {
 	s.Text = text
 }
 
-func getUpdateColumns(t table.Table, assignList []expressions.Assignment, multipleTable bool) ([]*column.Col, error) {
+func getUpdateColumns(t table.Table, assignList []expressions.Assignment, isMultipleTable bool) ([]*column.Col, error) {
 	// TODO: We should check the validate if assignList in somewhere else. Maybe in building plan.
 	tcols := make([]*column.Col, 0, len(assignList))
 	tname := t.TableName()
 	for _, asgn := range assignList {
-		if multipleTable {
+		if isMultipleTable {
 			if !strings.EqualFold(tname.O, asgn.TableName) {
 				continue
 			}
 		}
 		col := column.FindCol(t.Cols(), asgn.ColName)
 		if col == nil {
```

Comment:
The TableRefs may contains multiple table. So not all the column mentioned in assignList are in table t.